Mathlib/Algebra/GroupWithZero/WithZero.lean

```diff
@@ -266,6 +266,12 @@ def unitsWithZeroEquiv : (WithZero α)ˣ ≃* α where
   right_inv _ := rfl
   map_mul' _ _ := coe_inj.mp <| by simp only [Units.val_mul, coe_unzero, coe_mul]
 
+/- Importing `Mathlib.Algebra.Equiv.TransferInstance` would allow avoiding the `.surjective` call
+  but it would create too large an input, detected by the `asserts_not_exist` at the beginning of
+  this file. -/
```

Comment:
I think the answer here is that `Mathlib.Algebra.Equiv.TransferInstance` is a silly file that should be split

Reply:
I agree, but splitting that one is beyond the scope of this PR, I guess. I think the small comment is still relevant, but if you object against, I can remove it.

Reply:
I do think it doesn't add much value